src/components/pages/StudentsPage/WelcomeMessage.tsx

```diff
@@ -0,0 +1,29 @@
+/** @jsxImportSource @emotion/react */
+
+import { Box, Typography, Link } from '@mui/material';
+
```

Comment:
Lets use the `Link` from our shared components folder
```suggestion
import { Box, Typography } from '@mui/material';

import Link from '@components/shared/Link';
```